src/puppetlabs/puppetdb/export.clj

```diff
@@ -92,8 +91,14 @@
                                            query->wire-fn
                                            (maybe-anonymize anonymize-fn anon-config)
                                            (export-data->tar-items entity)
-                                           (add-tar-entries tar-writer)))]]
-      (query-fn query-api-version ["from" entity] nil query-callback-fn))))
+                                           (add-tar-entries tar-writer)))
+                  entity* (if (and (= entity "catalogs")
+                                   @storage/store-catalogs-jsonb-columns?)
+                            ;; *Warning* this can only be used in PE so it
+                            ;; *cannot be tested against in the FOSS repo
+                            "historical_catalogs"
```

Comment:
Yeah I don't exactly what we'd want to do here, maybe make an atom that PE can use to change the query entities or such eventually.